.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.3
```

Comment:
Don't include a .ruby-version file. As a library even contributors shouldn't be tied to a ruby version